src/pages/admin/Announcements.tsx

```diff
@@ -0,0 +1,960 @@
+  const RenderAnnouncementItem = ({
+    announcement,
+  }: AnnouncementProps): JSX.Element => {
+    return (
+      <div
+        className="announcementBox"
+        onClick={(e) => {
+          e.stopPropagation();
+          setCurrentAnnouncement(announcement);
+          setOpenEditModal(true);
+        }}
+      >
+        <h5>{announcement.title}</h5>
+        <div className="announcementIcons">
+          <img
+            className="addAnnouncementButton"
+            onClick={(e) => {
+              e.stopPropagation();
+              if (
+                confirm(
+                  'Are you sure you want to delete all instances of this announcement?'
+                )
+              ) {
+                deleteAnnouncement(announcement);
+              } else {
+              }
+            }}
+            width={33}
+            height={33}
+            alt="DeleteAnnouncementIcon"
+            src="/static/svg/Minus_Icon.svg"
+          ></img>
+        </div>
+        <div>
+          <div className="announcementSummary">
+            <p>
+              <b>Announcement Date:</b>
+              {announcement.publishedDate}
+              <br></br>
+              <b>Expiration Date:</b>
+              {announcement.expirationDate}
+              <br></br>
+              <b>Parish:</b>
+              {announcement.parish}
+            </p>
+          </div>
+        </div>
+      </div>
+    );
+  };
+  /* ============================= List of announcements ============================= */
+  const RenderAnnouncementList = (): JSX.Element => {
+    const announcementsLength: number = announcements?.filter(
+      (a: AnnouncementData) => {
+        if (a.parish === locationFilter) {
+          return true;
+        } else return false;
+      }
+    )?.length;
+    return (
+      <>
+        {announcements
+          .filter((a: AnnouncementData) => {
+            if (a.parish === locationFilter) {
+              return true;
+            } else return false;
+          })
+          .map((announcement: AnnouncementData, index: number) => {
+            if (index < count)
+              return (
+                <RenderAnnouncementItem
+                  key={announcement.id}
+                  announcement={announcement}
+                ></RenderAnnouncementItem>
+              );
+            else return null;
+          })}
+        {announcementsLength === 0 ? <h2>No Announcements found</h2> : null}
+        {announcementsLength &&
+        announcementsLength > 0 &&
+        announcementsLength > count ? (
+          <div
+            style={{
+              textAlign: 'center',
+              marginLeft: 'auto',
+              marginRight: 'auto',
+            }}
+          >
+            <button onClick={() => setCount(count + 5)}>Load More</button>
+          </div>
+        ) : null}
+      </>
+    );
+  };
+
+  const imageHelper = (announcement: AnnouncementData, conditional: string) => {
```

Comment:
You could make `conditional` a boolean since you always convert the data from boolean to string when `imageHelper()` is called.